change/@internal-calling-stateful-client-c3355bc5-9d82-4dca-8a07-ce9d1d0f8798.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
are these prerelease? As the change executes on stable builds

Reply:
Shouldn't this be none?
Doesn't affect public api